mods/ra/rules/defaults.yaml

```diff
@@ -296,6 +297,7 @@
 		MaxMoveDelay: 750
 
 ^ArmedCivilian:
+	Huntable:
```

Comment:
You don't need this, `^ArmedCivilian` is a template extension of `CivInfantry`, and every user of this template will have the `Huntable` from `Soldier` already.